tardis/io/tests/test_configuration_namespace.py

```diff
@@ -15,8 +15,13 @@ def data_path(filename):
     return os.path.join(data_dir, "data", filename)
 
 
-def test_simple_configuration_namespace():
-    config_ns = ConfigurationNameSpace(simple_config_dict)
+@pytest.fixture(scope="function")
```

Comment:
If we change the scope to something else, like "module"- tests would fail because the properties of the fixture are changed in multiple tests.
Since we do not want one test to affect the other, in this case, changing the scope to function would ensure that each test function gets a new instance of the `ConfigurationNameSpace` class (through the fixture).